src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Aerospike/AerospikeCommon.cs

```diff
@@ -0,0 +1,86 @@
+// <copyright file="AerospikeCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Tagging;
+using Datadog.Trace.Vendors.Serilog;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Aerospike
+{
+    internal class AerospikeCommon
+    {
+        private const string ServiceName = "aerospike";
+        private const string OperationName = "aerospike.command";
+        public const string IntegrationName = nameof(IntegrationIds.Aerospike);
+        internal static readonly IntegrationInfo IntegrationId = IntegrationRegistry.GetIntegrationInfo(IntegrationName);
+
+        public static Scope CreateScope<TTarget>(Tracer tracer, out AerospikeTags tags, TTarget target)
+        {
+            tags = null;
+
+            if (!tracer.Settings.IsIntegrationEnabled(IntegrationId))
+            {
+                // integration disabled, don't create a scope, skip this trace
+                return null;
+            }
+
+            Scope scope = null;
+
+            try
+            {
+                tags = new AerospikeTags();
+                var serviceName = tracer.Settings.GetServiceName(tracer, ServiceName);
+                scope = tracer.StartActiveWithTags(OperationName, tags: tags, serviceName: serviceName);
+                var span = scope.Span;
+
+                if (target.TryDuckCast<HasKey>(out var key) && key.Key != null)
+                {
+                    tags.Key = key.Key.ToString();
```

Comment:
Are there other tags that could be useful for users like the server name or address? Are there command types like insert, update, delete?

Reply:
I just saw `Write`, `Read`, `Exists`, and `Delete` in the test app.)

Reply:
That's hard to tell since I'm not familiar with aerospike 😄 The client talks to a cluster and I don't think I can extract what server in the cluster has accepted the request 🤔

Reply:
Discussed offline. Looks like it's not practical for now to iterate through the list of nodes to build some time of "remote host" information. We can come back and add it later if needed.